schemas/clinvar/tables/VariationArchiveVariation.json

```diff
@@ -0,0 +1,60 @@
+ {
+    "name": "variation_archive_variation",
+    "columns": [
+	{
+	    "name": "variation_archive_variation_id",
+	    "datatype": "int",
+	    "links": [
+		{
+		    "table_name": "variation_archive_variation",
+		    "column_name": "parent_id"
+		},
+		{
+		    "table_name": "clinical_assertion",
+		    "column_name": "variation_archive_variation_id"
+		}
+	    ]
+	},
+	{
+	    "name": "name",
+	    "datatype": "string"
+	},
+	{
+	    "name": "variation_type",
+	    "datatype": "string"
+	},
+	{
+	    "name": "subclass_type",
+	    "datatype": "string"
+	},
+	{
+	    "name": "allele_id",
+	    "datatype": "int"
+	},
+	{
+	    "name": "protein_change",
+	    "datatype": "string",
+	    "array_of": true
+	},
+	{
+	    "name": "num_copies",
+	    "datatype": "int"
+	},
+	{
+	    "name": "content",
+	    "datatype": "string"
+	},
+	{
+	    "name": "parent_id",
+	    "datatype": "int"
+	},
+	{
+	    "name": "variation_archive_id",
```

Comment:
`links`?

Reply:
According to @thathert this should be named something other than relationship or references as previously suggest to not be confused with the relationships field in the final JSON. It was suggest to used links. Please let me @danxmoran if I should change this.